internal/setup/yaml.go

```diff
@@ -25,6 +25,12 @@ type yamlRelease struct {
 	Format   string                 `yaml:"format"`
 	Archives map[string]yamlArchive `yaml:"archives"`
 	PubKeys  map[string]yamlPubKey  `yaml:"public-keys"`
+	// "v2-archives" is used for backwards compatibility with Chisel <= 1.0.0
+	// since it will be ignored. In new versions, it will be parsed with the new
+	// fields that break said compatibility, e.g. "pro" archives.
+	//
+	// Note: "archives" and "v2-archives" are merged together while parsing.
```

Comment:
The comment needs tuning as it's currently saying that it will both be ignored, in the very first sentence, and it will be merged together, at the end. It needs to s/v1.0.0 since/v1.0.0, where/, or similar.